nats/js/manager.py

```diff
@@ -197,34 +189,11 @@ async def delete_consumer(self, stream=None, consumer=None):
         )
         return resp['success']
 
-    async def get_last_msg(
```

Comment:
I've moved it into `JetStreamContext` because it uses `self._jsm`

Reply:
sounds good, thanks!